src/passes/referencedLibraries.ts

```diff
@@ -0,0 +1,34 @@
+import assert = require('assert');
+import { ContractDefinition, ContractKind, FunctionCall, MemberAccess } from 'solc-typed-ast';
+import { AST } from '../ast/ast';
+import { ASTMapper } from '../ast/mapper';
+
+export class ReferencedLibraries extends ASTMapper {
+  visitFunctionCall(node: FunctionCall, ast: AST): void {
+    if (node.vExpression instanceof MemberAccess) {
+      const contractDef = new Map();
+
+      node.vExpression.context?.map.forEach((astNode, key) => {
```

Comment:
It's best to use `ast.context` in situations where a context is needed to remove the possibility of it being undefined